### ISSUE TITLE
error: unable to invoke subcommand: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-xcode (No such file or directory)

### ISSUE THREAD

**Issue by IanGoodwin7** [appspector/URLSessionWebSocketTask-demo#1]
Hi 

Tried building this but I get 

error: unable to invoke subcommand: /Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift-xcode (No such file or directory)

Parent directory exists 
/Applications/Xcode-beta.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ 

Not in previous Xcode either, any ideas?

Regards

Ian G

**Comment by Lavallette** [comment]
I have the same problem. Did you find a solution? Tried to compile with Xcode 11 GM2 and have no idea how to solve this.

**Comment by IanGoodwin7** [comment]
no, I built a clean project based on IOS13 beta and it worked fine

**Comment by wuf810** [comment]
I've got this error too. Anyone have any idea how to get this working. Would love to try the demo app.
